引用了antd@5.7.x的项目在typescript编译时发生错误

### Reproduction link

[https://gitee.com/bahborben/antd-ecs](https://gitee.com/bahborben/antd-ecs)

### Steps to reproduce

环境： typescript 4.9.5

运行 "tsc -d"

控制台输出编译错误如下：

node_modules/antd/es/notification/PurePanel.d.ts:25:16 - error TS2304: Cannot find name 'PurePanel'.

25 export default PurePanel;
                  ~~~~~~~~~

src/comp/form/BaseForm.tsx:170:74 - error TS7006: Parameter 'e' implicitly has an 'any' type.

170             {undefined === props.onCancel ? undefined : <Button onClick={e => (props.onCancel as () => void)()}>{props.cancelTitle || i18n.t("form.BaseForm.cancel")}</Button>}
                                                                             ~

src/comp/form/BaseForm.tsx:171:50 - error TS7006: Parameter 'e' implicitly has an 'any' type.

171             {props.allowReset ? <Button onClick={e => handleReset()}>{props.resetTitle || i18n.t("form.BaseForm.reset")}</Button> : undefined}
                                                     ~

src/comp/table/ControlPanel.tsx:56:88 - error TS7006: Parameter 'e' implicitly has an 'any' type.

56               <Button type="primary" shape="circle" icon={<SearchOutlined />} onClick={e => {
                                                                                          ~

src/comp/table/ControlPanel.tsx:94:35 - error TS7006: Parameter 'e' implicitly has an 'any' type.

94                 (<Button onClick={e => setCommonFilterMode(false)}>more</Button>)
                                     ~

src/comp/table/ControlPanel.tsx:113:84 - error TS7006: Parameter 'e' implicitly has an 'any' type.

113             extraOperations={getCommonFilterItems().length > 0 ? [<Button onClick={e => setCommonFilterMode(true)}>back</Button>] : []}
                                                                                       ~


Found 6 errors in 3 files.

### What is expected?

编译通过

### What is actually happening?

控制台输出编译错误如下：

node_modules/antd/es/notification/PurePanel.d.ts:25:16 - error TS2304: Cannot find name 'PurePanel'.

25 export default PurePanel;
                  ~~~~~~~~~

src/comp/form/BaseForm.tsx:170:74 - error TS7006: Parameter 'e' implicitly has an 'any' type.

170             {undefined === props.onCancel ? undefined : <Button onClick={e => (props.onCancel as () => void)()}>{props.cancelTitle || i18n.t("form.BaseForm.cancel")}</Button>}
                                                                             ~

src/comp/form/BaseForm.tsx:171:50 - error TS7006: Parameter 'e' implicitly has an 'any' type.

171             {props.allowReset ? <Button onClick={e => handleReset()}>{props.resetTitle || i18n.t("form.BaseForm.reset")}</Button> : undefined}
                                                     ~

src/comp/table/ControlPanel.tsx:56:88 - error TS7006: Parameter 'e' implicitly has an 'any' type.

56               <Button type="primary" shape="circle" icon={<SearchOutlined />} onClick={e => {
                                                                                          ~

src/comp/table/ControlPanel.tsx:94:35 - error TS7006: Parameter 'e' implicitly has an 'any' type.

94                 (<Button onClick={e => setCommonFilterMode(false)}>more</Button>)
                                     ~

src/comp/table/ControlPanel.tsx:113:84 - error TS7006: Parameter 'e' implicitly has an 'any' type.

113             extraOperations={getCommonFilterItems().length > 0 ? [<Button onClick={e => setCommonFilterMode(true)}>back</Button>] : []}
                                                                                       ~


Found 6 errors in 3 files.

| Environment | Info |
| --- | --- |
| antd | 5.7.0 |
| React | 18.2.0 |
| System | Fedora Linux 38 (Workstation Edition) |
| Browser | chrome |

<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/components/notification/PurePanel.tsx
+++ b/components/notification/PurePanel.tsx
@@ -76,7 +76,7 @@ export interface PurePanelProps
   prefixCls?: string;
 }
 
-/** @internal Internal Component. Do not use in your production. */
+/** @private Internal Component. Do not use in your production. */
 const PurePanel: React.FC<PurePanelProps> = (props) => {
   const {
     prefixCls: staticPrefixCls,

